target/classes/META-INF/ejb-jar.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ejb-jar xmlns="http://java.sun.com/xml/ns/javaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://java.sun.com/xml/ns/javaee
+		  http://java.sun.com/xml/ns/javaee/ejb-jar_3_1.xsd"
+         version="3.1">
+    <enterprise-beans>
+        <entity id="CategoryEntity">
+            <ejb-name>CategoryEntity</ejb-name>
+            <ejb-class>SYSC4806.Model.Category</ejb-class>
+        </entity>
+        <entity id="CourseEntity">
+            <ejb-name>CourseEntity</ejb-name>
+            <ejb-class>SYSC4806.Model.Course</ejb-class>
+        </entity>
+        <entity id="LearningOutcomeEntity">
+            <ejb-name>LearningOutcomeEntity</ejb-name>
+            <ejb-class>SYSC4806.Model.LearningOutcome</ejb-class>
+        </entity>
+        <entity id="ProgramEntity">
+            <ejb-name>ProgramEntity</ejb-name>
+            <ejb-class>SYSC4806.Model.Program</ejb-class>
+        </entity>
+    </enterprise-beans>
```

Comment:
are you just re-adding this file and persistence.xml? (sorry, just not sure whats going on here :/)

Reply:
Umm, sorry I think we are due for another issue! I'm pretty sure we should not be committing any of the "target/" files. In a standard repository, we should not be committing compiled files and compiled binaries.

I believe the .gitignore isn't sufficient enough, since we should also be ignoring the standard stuff for a Maven project. Our spring-boot framework is using a maven project :).

Raised the issue in #48